src/settings/menuoptions.h

```diff
@@ -54,6 +54,7 @@ class MenuOptions : public QDialog
     int mMenuHeight             = 280;
 private slots:
     void onBtnClicked(QAbstractButton * button);
```

Comment:
void onBtnClicked(QAbstractButton *button);

Reply:
it is Roman's code